### PR TITLE
Added Text- Utility compoents

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Added heading usable component
-> Added small text usable component
-> Added grey text usable component
-> Added centre text usable components

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
good to merge

File: text-utility/text-utility.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+.h4-head {
+  font-size: 1.75rem;
+}
+
+.h5-head {
+  font-size: 1.5rem;
+}
+
+.lg-text {
```
Comment (lannisterrr) [inline text-utility/text-utility.css:31]: You can also font-weight and line-height for the text .

File: text-utility/text-utility.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.head {
+  color: #000;
```
Comment (way2nizam) [inline text-utility/text-utility.css:2]: yes, you can define font-weight here for headings.

File: text-utility/text-utility.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.h5-head {
+  font-size: 1.5rem;
+}
+
+.lg-text {
+  font-size: 1.5rem;
+  line-height: 1.5;
+  word-break: break-all;
```
Comment (way2nizam) [inline text-utility/text-utility.css:34]: for word-break, you can add a variable in the typography section and use this property with the variable name like color.

**Review by way2nizam — COMMENTED** [review]
added some suggestions.

File: text-utility/text-utility.css (hunk 1 above)
Comment (ashishnnnnn) [reply text-utility/text-utility.css:31]: Done.. For heading i have added the font-weight separately..

File: text-utility/text-utility.css (hunk 2 above)
Comment (ashishnnnnn) [reply text-utility/text-utility.css:2]: Done.. For heading i have added the font-weight separately..

File: text-utility/text-utility.css (hunk 3 above)
Comment (ashishnnnnn) [reply text-utility/text-utility.css:34]: Yaa.. we could do that.. but as typography css is small in terms of code line.. so i thought to put it individually ..

Comment (way2nizam) [reply text-utility/text-utility.css:34]: okay. you can merge.

File: text-utility/text-utility.css (hunk 2 above)
Comment (way2nizam) [reply text-utility/text-utility.css:2]: okay

**Pull request merged**